pkg/agent/route/route_windows.go

```diff
@@ -174,6 +191,103 @@ func (c *Client) DeleteRoutes(podCIDR *net.IPNet) error {
 	return nil
 }
 
+func (c *Client) AddServiceVirtualIPToGWRoutes() {
+	metric := util.MetricHigh
+	linkIndex := c.nodeConfig.GatewayConfig.LinkIndex
+	gwIP := c.nodeConfig.GatewayConfig.IPv4
+
+	// A virtual IP is used to avoid too many ARP responders in OVS flows.
+	// Route: Virtual IP (169.254.169.252) -> antrea-gw0
+	vRoute := &util.Route{
+		LinkIndex:         linkIndex,
+		DestinationSubnet: &serviceGWVirtualIPNet,
+		GatewayAddress:    gwIP,
+		RouteMetric:       metric,
+	}
+	util.NewNetRoute(vRoute)
+
+	// For ClusterIP Service from host, reply traffic needs a route entry to route packet to vIP via antrea-gw0.
```

Comment:
vIP -> ServiceGWVirtualIP

Reply:
Updated.

Reply:
Did not see it is updated.

Reply:
And: For ClusterIP Service from host -> For ClusterIP Service requests from host

Reply:
Sorry I missed this one. Updated.